### PR TITLE
Style adjustment

### PR DESCRIPTION
**Pull request by Maenhout**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import styled from 'styled-components';
 
 const SShadow = styled.div`
-  box-shadow: 0 50vh 0 50vh #222222;
+  /* box-shadow: 0 50vh 0 50vh #222222; */
```
Comment (GeorgesFORTRY) [inline src/components/Footer.jsx]: death code

**Review by GeorgesFORTRY — COMMENTED** [review]
remove death code

**Review by GeorgesFORTRY — APPROVED** [review]
it's ok !

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**